src/librustdoc/passes/stripper.rs

```diff
@@ -125,8 +125,7 @@ crate struct ImplStripper<'a> {
 impl<'a> DocFolder for ImplStripper<'a> {
     fn fold_item(&mut self, i: Item) -> Option<Item> {
         if let clean::ImplItem(ref imp) = *i.kind {
-            // emptied none trait impls can be stripped
-            if imp.trait_.is_none() && imp.items.is_empty() {
+            if imp.trait_.is_none() && imp.items.is_empty() && i.doc_value().is_none() {
```

Comment:
Why get rid of the comment? I think it was helpful. We should update to "Impls can be skipped if they are: empty; not a trait impl; and have no documentation."